### ISSUE TITLE
Allow multiple authors

### ISSUE THREAD

**Issue by dstansby** [UCL-ARC/python-tooling#295]
### Is Your Feature Request Related to a Problem? Please Describe

When the cookiecutter is run, it asks for `author_given_names` and `author_family_names`. This doesn't allow for the possibility of the software having multiple authors however.

### Describe the Solution You'd Like

An option for multiple authors to be added when creating a new package.

**Comment by samcunliffe** [comment]
I remember discussing this and we decided that one author was the least of all evils.
The problem with multiple authors, is that there's not nice syntax to do this.

The options [AFAIR](https://en.wiktionary.org/wiki/AFAIR):
* Take lists of names and emails, then split (which is problematic if you have a different name order or multiple surnames or... anything vaguely non-John-Smith).
* Have `author_2_firstname` and `author_2_secondname` etc. Then cancel all after an empty response is recieved.

[Here's a comment where we discussed it.](https://github.com/UCL-ARC/python-tooling/issues/22#issuecomment-1649661479)

**Comment by paddyroddy** [comment]
Yeah, it becomes quite non-trivial to support n-authors. Hard to know what is best to do here.
